README.md

```diff
@@ -95,6 +92,40 @@ A example setup, which defines an HTTPS proxy and its self-signed certificate, w
     # REMOVE following option when using it on Windows (see above)
     caBundleFile /path/to/proxy-certificate-bundle.pem
 ```
+#### Certificates
+Certificates can be trick because depends on the proxy (`HTTP` vs `HTTPS`), Linux, Windows, Logging Endpoint,
+Infra-Agent Endpoint, etc. We will talk about these configurations below.
+
+##### Linux
+For `HTTP` proxies we don't need to setup any certificate. Out of the box the plugin will load the system certificates
+and we will be able to send logs into the logging endpoint.
+
+For `HTTPS` proxies you can specify the self-signed certificate (PEM file) using either the `caBundleFile` or `caBundleDir`
+parameters.
+
+##### Windows
+Similar to Linux, for `HTTP` proxies we don't need to setup any certificate. Out of the box the plugin will load
+the system certificates.
+
+For `HTTPS` it require attention:
+
+The recommended process to import the self-signed certificate (PEM file) using the MMC tool. You can refer to
+[this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), but in Step 2
+ensure to import it in your "Trusted Root Certification Authorities" instead of importing it in the
+"Intermediate Certification Authorities".
```

Comment:
```suggestion
The recommended process to import the proxy self-signed certificate (PEM file) using the MMC tool. You can refer to
[this link](https://www.ssls.com/knowledgebase/how-to-import-intermediate-and-root-certificates-via-mmc/), but in Step 2
ensure to import it in your `Trusted Root Certification Authorities` instead of importing it in the
`Intermediate Certification Authorities`.
```